client/my-sites/subscribers/controller.js

```diff
@@ -26,6 +27,24 @@ const queryStringChanged = ( key ) => ( value ) => {
 	return page.show( addQueryArgs( path, { [ key ]: value } ) );
 };
 
+const addSubscribersDataviewsClassName = () => {
```

Comment:
I'm adding the body class to style the layout. Seems a bit hacky but I found it being added this way elsewhere.